sdk/cognitiveservices/Azure.InkRecognizer.UWP.Stroke/samples/App.xaml

```diff
@@ -0,0 +1,7 @@
+﻿<Application
```

Comment:
Please rename this directory to `sdk/cognitiveservices/InkRecognizer.WPF.Stroke` You should remove Azure from it because it is redundant and inconsistent with the others.

Reply:
Done. Thanks .